pwnlib/elf/corefile.py

```diff
@@ -1109,11 +1177,8 @@ def registers(self):
 
         return rv
 
-    def debug(self, *a, **kw):
```

Comment:
LOL

Reply:
Yeah I'm not sure why we ever had *a **kw and then immediately throw an error if you actually pass in args.  Was probably trying to match some other prototype, no idea how this ever made it into the code.